blocks/library/button/index.js

```diff
@@ -130,6 +119,18 @@ registerBlockType( 'core/button', {
 					</InspectorControls>
 				}
 			</span>,
+			focus && (
+				<form
+					className="blocks-format-toolbar__inline-link-modal"
+					onSubmit={ ( event ) => event.preventDefault() }>
+					<Dashicon icon="admin-links" />
+					<UrlInput
+						value={ url }
+						onChange={ ( value ) => setAttributes( { url: value } ) }
+					/>
+					<IconButton icon="editor-break" label={ __( 'Apply' ) } type="submit" />
+				</form>
+			)
```

Comment:
@jasmussen Yep, travis is failing because of a missing trailing comma after this parathensis. You should definitely install an `eslint` plugin to your favorite IDEE ;)

Reply:
💓💓👍🌌

Reply:
Have a nice weekend Joen :)